server/hap2/hatohol/hap2_zabbix_api.py

```diff
@@ -111,6 +111,8 @@ def update_events(self, last_info=None, count=None, direction="ASC",
 
         event_id_from = None
         if direction == "ASC":
+            if isinstance(last_info, unicode) or isinstance(last_info, str):
+                last_info = int(last_info)
             if isinstance(last_info, int):
```

Comment:
Sorry, I forgot in case of event_from is null.
This code is not necessary to remove.